main_he.tex

```diff
@@ -0,0 +1,61 @@
+% !TeX spellcheck = en_US
+\documentclass[12pt]{article}
+
+
+% Language setting
+\usepackage{fontspec} % For font settings
+\usepackage{polyglossia} % Better for handling multiple languages with XeLaTeX or LuaLaTeX
+
+% Set languages
+\setmainlanguage{hebrew} % Set the main language to Hebrew
+\setotherlanguage{english} % Optional: if you have English text as well
+
+% Set main font for Latin script (English)
+\setmainfont{Times New Roman}
```

Comment:
Why? Original LaTeX font doesn't work?

Reply:
It didn't compiled for me without it (also it didn't work without \setmonofont{DejaVu Sans Mono})

I don't mind reworking on this file later on. for now I just wanted it to compile so I'll see the pdf and see what is done and what to fix in the translation itself